agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -4706,6 +4708,14 @@ bool slave_thread::handle_client_capability_query(Socket *sd, ieee1905_1::CmduMe
     return true;
 }
 
+bool slave_thread::handle_ap_metrics_query(Socket *sd, ieee1905_1::CmduMessageRx &cmdu_rx)
+{
+    const auto mid = cmdu_rx.getMessageId();
+    LOG(DEBUG) << "Received AP_METRICS_QUERY_MESSAGE , mid=" << std::dec << int(mid);
```

Comment:
I think it is better to use std::hex for the mid. I will only have 4 figures that way.